src/exercise/client/src/Components/FilterBar/FilterBar.js

```diff
@@ -0,0 +1,91 @@
+import "./FilterBar.css";
+import { Dropdown, Search } from "monday-ui-react-core";
+import { useState } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "@reduxjs/toolkit";
+import { getTaskList } from "../../redux/selectors/taskListSelectors";
+import { updateFilterTasksAction } from "../../redux/actions/filterTasksActions";
+import { setTasksAction } from "../../redux/actions/taskActions";
+
+function SearchBar({
+  tasks,
+  setTasksAction,
+  updateFilterTasksAction,
+  setSearchFilter,
+}) {
+  const [clicked, setClicked] = useState(false);
+
+  function handleFilter(value) {
+    if (value) {
+      let tmp = [...tasks];
+      switch (value.label) {
+        case "All":
+          updateFilterTasksAction(tmp);
+          setTasksAction(tasks);
+          return;
+        case "To Do":
+          tmp = tmp.filter((task) => task.status === false);
+          updateFilterTasksAction(tmp);
+          setTasksAction(tasks);
+          return;
+        case "Completed":
+          tmp = tmp.filter((task) => task.status === true);
+          updateFilterTasksAction(tmp);
+          setTasksAction(tasks);
+          return;
+        default:
+          return;
+      }
+    }
+  }
+
+  return (
+    <div className="filterBar">
+      <Search
+        placeholder="Search Task"
+        onFocus={() => setClicked(true)}
```

Comment:
you can perform this CSS behavior using only CSS (and it's also recommended)
using the :focus modifier